### PR TITLE
Misc updates, release notes

### PR DESCRIPTION
**Pull request by wteouchicago**
- added blank lines so bullets render correctly
- adding prelim release notes > known issues, including DOC-401/DOC-346

### CODE REVIEW

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md [hunk 1]
```diff
@@ -68,11 +68,35 @@ The GDC Data Portal has been completely revamped to include the following featur
 
 ### Known Issues and Workarounds
 
+* Cohorts:
+    * Cohorts are under active development and their behavior may change in the first several months after the release of GDC Portal 2.0. As this process may result in the loss of saved cohorts on the portal, we highly recommend [exporting cohorts](quick_start.md#main-toolbar) locally.
```
Comment (wwysoc2) [inline docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:72]: I can't find any release notes examples, but I think the link needs to be /Data_Portal/Users_Guide/quick_start.md#main-toolbar

**Review by wwysoc2 — APPROVED** [review]
Looks good except for the link.  Feel free to change it or merge it.  I can play around with the link to verify that it works.

File: docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md (hunk 1 above)
Comment (wteouchicago) [reply docs/Data_Portal/Release_Notes/Data_Portal_Release_Notes.md:72]: oh good point. i forgot i was writing release notes and not user's guide... :P will fix.

**Comment by wteouchicago** [comment]
merging so i can start a new branch off develop-v2

**Pull request merged**